### ISSUE TITLE
all bibliogram instances non-working - alternative...

### ISSUE THREAD

**Issue by atomGit** [SimonBrazell/privacy-redirect#277]
imginn.org - could this be a viable alternative?

format is` imginn.org/<user_name>/`
`https://imginn.org/maria.zeee/`

**Comment by pverton** [comment]
Agreed that an alternative is needed. No idea how to determine if it's a valid alternative though.
